introduction.md

```diff
@@ -0,0 +1,3 @@
+Hi. I am Ezequiel. I am following the mastering github course in codeschool.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.